libs/shared/ui/nuverial/src/lib/components/form-errors/form-errors.component.spec.ts

```
@@ -67,4 +67,55 @@ describe('FormFieldErrorComponent', () => {
 
     expect(component.formErrorClick.emit).toHaveBeenCalledWith(formErrors[0]);
   });
+
+  it('should not display "SHOW MORE" button when formErrors array length is 0', async () => {
+    await getFixtureByTemplate();
+
+    expect(screen.queryByText('SHOW MORE')).toBeNull();
+  });
+
+  it('should show "SHOW MORE" link when there are more than 3 errors', async () => {
+    const { fixture } = await getFixture({
+      componentProperties: {
+        formErrors: [...formErrors, ...formErrors, ...formErrors, ...formErrors],
+      },
+    });
+
+    const showMoreLink = fixture.nativeElement.querySelector('.show-more a');
+    expect(showMoreLink).toBeTruthy();
+    expect(showMoreLink.textContent.trim()).toBe('SHOW MORE');
+  });
+
+  it('should show additional errors when "SHOW MORE" link is clicked', async () => {
+    const { fixture } = await getFixture({
+      componentProperties: {
+        formErrors: [...formErrors, ...formErrors, ...formErrors, ...formErrors],
+      },
+    });
+
+    const showMoreLink = fixture.nativeElement.querySelector('.show-more a');
+    showMoreLink.click();
+
+    fixture.detectChanges();
+
+    const additionalErrors = fixture.nativeElement.querySelectorAll('li');
+    expect(additionalErrors.length).toBeGreaterThan(0);
+  });
+
+  it('should show "SHOW LESS" link when additional errors are displayed', async () => {
+    const { fixture } = await getFixture({
+      componentProperties: {
+        formErrors: [...formErrors, ...formErrors, ...formErrors, ...formErrors],
+      },
+    });
+
+    const showMoreLink = fixture.nativeElement.querySelector('.show-more a');
+    showMoreLink.click();
+
+    fixture.detectChanges();
+
+    const showLessLink = fixture.nativeElement.querySelector('.show-more a');
+    expect(showLessLink).toBeTruthy();
+    expect(showLessLink.textContent.trim()).toBe('SHOW LESS');
```

You should add a similar assertion as in the above test to make sure that the extra errors are hidden. That is how you can test the actual behavior,